packages/material-ui/src/Stack/Stack.test.js

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import Stack from './Stack';
```

Comment:
We prefer testing the public API

```suggestion
import Stack from '@material-ui/core/Stack';
```